Storage/SQL/BrowserDB.swift

```diff
@@ -220,7 +219,9 @@ open class BrowserDB {
             }
             return success
         }) {
-            // Err getting a transaction
+            // Error getting a transaction
+            log.error("Unable to get a transaction: \(error.localizedDescription)")
+            SentryIntegration.shared.sendSync(message: error.localizedDescription, tag: "BrowserDB", severity: .error)
```

Comment:
There's a lot more I want to log in this method in general. Just want to get this landed for now.